source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,165 @@
+  of 16 megabytes.
+
+- Pipeline stages have a memory limit of 100 megabytes by default. If necessary, you may exceed this limit by setting the ``allowDiskUse``
+  property of ``AggregateOptions`` to ``true``. See the
+  :node-api-4.0:`AggregateOptions API documentation <interfaces/aggregateoptions.html>`
+  for more details.
+
+.. important:: ``$graphLookup`` exception
+
+   The :manual:`$graphLookup
+   </reference/operator/aggregation/graphLookup/>` stage has a strict
+   memory limit of 100 megabytes and will ignore ``allowDiskUse``.
+
+Useful References
+~~~~~~~~~~~~~~~~~
+
+- :manual:`Aggregation pipeline </core/aggregation-pipeline/>`
+- :manual:`Aggregation stages </meta/aggregation-quick-reference/#stages>`
+- :manual:`Operator expressions </meta/aggregation-quick-reference/#operator-expressions>`
+
+Runnable Examples
+-----------------
+
+Create a new Node.js file called ``AggTour.js`` and connect to your
+MongoDB deployment to follow this example.
+
+.. tip::
+
+   For information on connecting to your MongoDB deployment, see the :ref:`Connection Guide <connection>`.
+   
+Insert Sample Data
+~~~~~~~~~~~~~~~~~~
+
+The example uses a collection of documents corresponding
+to information about 10 dining establishments. Insert the sample data
+into the ``restaurants`` collection of the ``aggregationTest`` database:
+
+.. code-block:: javascript
+
+   const db = client.db("aggregationTest");
+   const coll = db.collection("restaurants");
+   
+   const docs = [
+        { "name": "Sun Bakery Trattoria", "stars": 4, "categories": ["Pizza", "Pasta", "Italian"] },
+        { "name": "Blue Bagels Grill", "stars": 3, "categories": ["Bagels", "Cookies", "Sandwiches"] },
+        { "name": "Hot Fresh Bagels", "stars": 4, "categories": ["Bagels", "Sandwiches", "Coffee"] },
+        { "name": "Cafe au Late", "stars": 4, "categories": ["Bakery", "Cafe", "Coffee", "Dessert"] },
+        { "name": "Green Feast Pizzeria", "stars": 2, "categories": ["Pizza", "Italian"] },
+        { "name": "Go Pasta", "stars": 0, "categories": ["Pasta", "Italian", "Cafeteria"] },
+        { "name": "Liz's Coffee Bar", "stars": 5, "categories": ["Coffee", "Cafe", "Bakery"] },
+        { "name": "Oak Steakhouse", "stars": 0, "categories": ["Steak", "Seafood"] },
+        { "name": "Petit Cookie", "stars": 4, "categories": ["Bakery", "Cookies", "Cake", "Coffee"] },
+        { "name": "Modern Eats", "stars": 3, "categories": ["Steak", "Salad", "American"] }
+    ];
+
+    // This option prevents additional documents from being inserted if one fails.
+    const options = { ordered: true };
+
+    const result = await coll.insertMany(docs, options);
+
+Aggregation Example
+~~~~~~~~~~~~~~~~~~~
+
+To perform an aggregation, pass a list of aggregation stages to the 
+``collection.aggregate()`` method.
+
+In the following example, the aggregation pipeline makes use of the
```

Comment:
```suggestion
In the following example, the aggregation pipeline uses the
```

I think "makes use" is idiomatic and may confuse non-native speakers. [See the style guide entry on writing for a global audience](https://docs.mongodb.com/meta/style-guide/terminology/terms-for-global-audience/#don-t-use-idioms-or-colloquialisms) for a full write up on this.

Reply:
accepted